modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc

```diff
@@ -83,7 +83,7 @@ The full list of conditions is (File mimetype upload):
 * ends with
 * doesn’t end with
 
-NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] that ownCloud supports is available the source of ownCloud core.
+NOTE: The complete list of available mimetypes which ownCloud supports is available {supported-mimetype-list-url}[in the ownCloud core source].
```

Comment:
Just to improve my English :smile:
Is it `in the ownCloud core source` or `in the source code of ownCloud core` ?

Reply:
They're, effectively, the same. The first is active tense, the second is passive.